lib/active_merchant/billing/gateways/hps.rb

```diff
@@ -124,7 +142,15 @@ def add_customer_data(xml, credit_card, options)
         end
       end
 
-      def add_payment(xml, card_or_token, options)
+      def add_check_customer_data(xml, check, options)
+        xml.hps :ConsumerInfo do
+          xml.hps :FirstName, check.first_name
+          xml.hps :LastName, check.last_name
+          xml.hps :CheckName, check.bank_name
```

Comment:
I might be reading this incorrectly, but this seems to be an option for setting a business name, not the name of the bank:
![image](https://user-images.githubusercontent.com/42787645/72743907-89abad80-3b7a-11ea-8d88-debbaa4eec78.png)

Reply:
I definitely thought I had seen evidence to the contrary elsewhere, but now I can't find it! 🤷‍♀